dev/e2e_app/integration_test/external_link_test.dart

```diff
@@ -7,7 +9,7 @@ void main() {
     await $.native.openUrl('https://leancode.co');
 
     try {
-      await $.native.tap(Selector(text: 'Use without an account'));
+      await $.native.tap(Selector(text: 'Contact us'));
```

Comment:
On first chrome open (e.g. on FTL device) it's ask to login or use without account